CONTRIBUTING.md

```diff
@@ -239,15 +241,15 @@ var p = new Promise(function () { /* some test code */ });
 
 p.then(function checkAssertions(arg) {
   if (!expected_condition) {
-    $ERROR("failure message");
+    throw new Test262Error("failure message");
   }
 
 }).then($DONE, $DONE);
 ```
 
 Function `checkAssertions` implicitly returns `undefined` if the expected condition is observed.  The return value of function `checkAssertions` is then used to asynchronously invoke the first function of the final `then` call, resulting in a call to `$DONE(undefined)`, which signals a passing test.
 
-If the expected condition is not observed, function `checkAssertions` throws a `Test262Error` via function $ERROR.  This is caught by the Promise and then used to asynchronously invoke the second function in the call -- which is also `$DONE` -- resulting in a call to `$DONE(error_object)`, which signals a failing test.
+If the expected condition is not observed, function `checkAssertions` throws a `Test262Error`.  This is caught by the Promise and then used to asynchronously invoke the second function in the call -- which is also `$DONE` -- resulting in a call to `$DONE(error_object)`, which signals a failing test.
```

Comment:
this description makes it compelling to leave the `throw new Test262Error` above, we can also consider a failing assertion will throw an exception, being caught by the Promise.